README.md

```diff
@@ -54,18 +56,19 @@ Last stable versions of packages that have been provided for usage with Azure an
 
 Libraries which enable you to provision specific resources. They are responsible for directly mirroring and consuming Azure service's REST endpoints. Management library directories contain `-mgmt-`, e.g. `azure-mgmt-keyvault`.
 
-## Need help?
+## Need help
 
-* For reference documentation visit the [Azure SDK for Java documentation](http://aka.ms/java-docs).
-* For tutorials, samples, quick starts and other documentation, visit [Azure for Java Developers](https://docs.microsoft.com/java/azure/).
-* For build reports on code quality, test coverage, etc, visit [Azure Java SDK](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/index.html).
-* File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-java/issues/new/choose).
-* Check [previous questions](https://stackoverflow.com/questions/tagged/azure-java-sdk) or ask new ones on StackOverflow using `azure-java-sdk` tag.
+- For reference documentation visit the [Azure SDK for Java documentation](http://aka.ms/java-docs).
+- For tutorials, samples, quick starts and other documentation, visit [Azure for Java Developers](https://docs.microsoft.com/java/azure/).
+- For build reports on code quality, test coverage, etc, visit [Azure Java SDK](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/index.html).
+- File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-java/issues/new/choose).
+- Check [previous questions](https://stackoverflow.com/questions/tagged/azure-java-sdk) or ask new ones on StackOverflow using `azure-java-sdk` tag.
 
 ## Navigating the repository
 
 ### Master branch
-The master branch has the most recent code with new features and bug fixes. It does **not** represent latest released **GA** SDK. See [above](#Client-GA-January-2020-Releases) for latest **GA** release.<br/>
+
+The master branch has the most recent code with new features and bug fixes. It does **not** represent latest released **GA** SDK. See [above](#Client-GA-January-2020-Releases) for latest **GA** release.
```

Comment:
Should this link be "Client-GA-February-2020-Releases"?

Reply:
Just pushed the fix for it, forgot to push my latest local revisions 😄